bin/fetch-proxy

```diff
@@ -1,18 +1,18 @@
-#!/bin/bash
+#!/bin/sh
 
 # If the first argument to this script is "latest" or unset, it fetches the
-# latest proxy binary via build.l5d.io/linkerd2-proxy/latest.txt. If it's set to
+# latest proxy binary from the linkerd2-proxy github releases. If it's set to
 # a commit sha from the master branch of the linkerd2-proxy repo, it will fetch
```

Comment:
I believe the first argument can be set to a proxy version number such as `v2.76.0` but not a commit sha.

Reply:
Done!